test/browser/commands.test.js

```diff
@@ -115,29 +165,33 @@ describe('commands', () => {
 
 	describe('mouseReset', () => {
 
+		let elem;
+		const buttonTemplate = html`<button>text</button>`;
 		const mousePos = { x: 0, y: 0 };
 		function onMouseMove(e) {
 			mousePos.x = e.clientX;
 			mousePos.y = e.clientY;
 		}
 
-		const buttonTemplate = html`<button>text</button>`;
+		before(() => {
+			window.addEventListener('mousemove', onMouseMove);
+		});
```

Comment:
Didn't see this stuff doing the same thing down here -- could just move it up under the `commands` fixture and then every test in the whole file can use it.